src/zzz_od/application/suibian_temple/operations/suibian_temple_auto_manage.py

```
@@ -0,0 +1,131 @@
+from one_dragon.utils.i18_utils import gt
+from zzz_od.context.zzz_context import ZContext
+from zzz_od.operation.zzz_operation import ZOperation
+
+
+class SuibianTempleAutoManage(ZOperation):
+
+    def __init__(self, ctx: ZContext):
+        """
+        随便观 - 自动托管
+
+        需要在随便观主界面时候调用，完成后返回随便观主界面
```

Ruff (0.14.6)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)